### ISSUE TITLE
Startup Script Requires the Working Directory to Match with the Project Folder

### ISSUE THREAD

**Issue by martinvahi** [HelloZeroNet/ZeroNet#569]
The following example shows, how the star-up of ZeroNet fails, if the `pwd`  is outside of the project home folder:

```
zeronet_runner@computenode1softf1com /media/usb0/large_files/zeronet_runner $ nice -n20 python ./ZeroNet/zeronet.py 
- Starting ZeroNet...
Traceback (most recent call last):
  File "./ZeroNet/zeronet.py", line 15, in main
    import main
  File "./ZeroNet/src/main.py", line 86, in <module>
    PluginManager.plugin_manager.loadPlugins()
  File "./ZeroNet/src/Plugin/PluginManager.py", line 28, in loadPlugins
    for dir_name in os.listdir(self.plugin_path):
OSError: [Errno 2] No such file or directory: 'plugins'
zeronet_runner@computenode1softf1com /media/usb0/large_files/zeronet_runner $ ls
ZeroNet  data  log
zeronet_runner@computenode1softf1com /media/usb0/large_files/zeronet_runner $ ls ./ZeroNet/
Dockerfile  README.md    plugins           src       tools      zeronet.py
LICENSE     Vagrantfile  requirements.txt  start.py  update.py
zeronet_runner@computenode1softf1com /media/usb0/large_files/zeronet_runner $ uname -a
Linux computenode1softf1com 4.1.19+ #858 Tue Mar 15 15:52:03 GMT 2016 armv6l GNU/Linux
zeronet_runner@computenode1softf1com /media/usb0/large_files/zeronet_runner $ date
Mon Sep  5 07:09:20 UTC 2016
zeronet_runner@computenode1softf1com /media/usb0/large_files/zeronet_runner $
```

**Comment by grez911** [comment]
@martinvahi Is this problem still present? What is your OS? I can't reproduce it on Ubuntu.

**Comment by HelloZeroNet** [comment]
Fixed 0.5.1: https://github.com/HelloZeroNet/ZeroNet/commit/8b7bd2d5720262273ccdf88e52835075b4460081
